features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -87,7 +88,8 @@ Feature: Collection
   Then I should see "Challenge was successfully created"
   When I log out
     And I am logged in as "myname1"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.